tasklist/client/src/common/form-js/FormJSRenderer/extractFilePath.test.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {extractFilePath} from './extractFilePath';
+
+describe('extractFilePath', () => {
+  it('should extract file paths from an object', () => {
+    const mock = {
+      a: 'files::123',
+      b: {
+        c: 'files::456',
+        d: 'not a file',
+      },
+      e: [
+        'files::789',
+        {
+          f: 'files::012',
+        },
+      ],
+    };
+
+    const result = extractFilePath(mock);
+
+    expect(result.size).toBe(4);
+    expect(result.get('files::123')).toBe('a');
+    expect(result.get('files::456')).toBe('b.c');
```

Comment:
It might be a good idea to adda test for one more level. we could test `b.c.x`

Reply:
Done